src/components/dashboard/components/DashboardLayout.js

```diff
@@ -1,11 +1,14 @@
 /* eslint-disable max-lines-per-function */
 import React from 'react';
 import './DashboardLayout.css';
-import { Layout } from 'antd';
-import { Route, Switch, useHistory } from 'react-router-dom';
+import { Layout, Breadcrumb } from 'antd';
+import {
+    Route, Switch, useHistory, Link, withRouter
+} from 'react-router-dom';
+
 import { useDispatch } from 'react-redux';
 import SideNav from './SideNav';
-import { LOGOUT } from '../constants';
+import { LOGOUT, BREADCRUMB_NAME_MAP } from '../constants';
 import CreateElection from '../../createElection/components';
 import { components as ViewElection } from '../../viewElection';
 import { components as ViewStats } from '../../viewStats';
```

Comment:
Something is off with the import order

Reply:
Use this guide recommended by Ola to fix this https://medium.com/dooboolab/using-eslint-prettier-and-sort-imports-vscode-extensions-for-formatting-open-source-project-16edf317129d